students/Tommy Aguilu/mailroom_part2.py

```diff
@@ -0,0 +1,65 @@
+### Author - Tommy Aguilu
```

Comment:
Update mailroom with file writing.
Goal: Write a full set of letters to all donors to individual files on disk.

Where is this accomplished?